backend-shared/plannershared.cpp

```diff
@@ -313,5 +313,10 @@ void plannerShared::set_display_variations(bool value)
 // Handle when user change length measurement type
 void plannerShared::unit_lengthChangedSlot(int value)
 {
-	qDebug() << "---> change METER <-> FEET";
+	// Provoke recalculation of model and send of signals
+	set_ascratelast6m(ascratelast6m());
+	set_ascratestops(ascratestops());
+	set_ascrate50(ascrate50());
+	set_ascrate75(ascrate75());
+	set_descrate(descrate());
```

Comment:
please collapse this with the previous commit and remove the `#include <QDebug>`

Reply:
done